pkg/controller/kieapp/webconsole/webconsole.go

```diff
@@ -0,0 +1,136 @@
+package webconsole
+
+//go:generate go run -mod=vendor ../defaults/.packr/packr.go
+
+import (
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/logs"
+	"github.com/kiegroup/kie-cloud-operator/pkg/controller/kieapp/webconsole/utils"
+	"strconv"
+	"strings"
+
+	consolev1 "github.com/openshift/api/console/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CustomResourceDefinition struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+}
+
+var log = logs.GetLogger("kieapp.webconsole")
+
+func CreateSampleYAMLs() {
+	log.Info("Loading Console yaml samples.")
+	files, err := loadFilesOnlyWithBox("consoleyamlsamples")
```

Comment:
This should use actual files in `deploy/crs`. I'm not sure what this currently does or denotes.

Reply:
currently for testing, now moved to crs folder.

Reply:
done